libraries/JGUZDV.Blazor.Components/src/Typescript/Modal.ts

```diff
@@ -0,0 +1,37 @@
+﻿export class closeListener {
+    constructor() {
+        this.handleCloseEvent = this.handleCloseEvent.bind(this);
+    };
+
+    public addListener(guid, reference) {
+        var dialog = document.getElementById(guid);
+            dialog.addEventListener("close", this.handleCloseEvent(reference));
+    }
+
+    public removeListener(guid, reference) {
+        var dialog = document.getElementById(guid);
+            dialog.removeEventListener("close", this.handleCloseEvent(reference));
+        
+     }
+
+    private handleCloseEvent(reference) {
+       return () => reference.invokeMethodAsync("Close");
+    }
+}
+
+const testClass = new closeListener();
```

Comment:
Ich denke, Testcode sollte schon gar nicht in einem Commit sein, aber spätestens im PR sollte es nicht auftauchen.